forge-gui/res/cardsfolder/c/chefs_kiss.txt

```diff
@@ -1,7 +1,8 @@
 Name:Chef's Kiss
 ManaCost:1 R R
 Types:Instant
-A:SP$ ControlSpell | ValidTgts$ Card | TgtPrompt$ Select target spell that targets only a single permanent or player | TargetType$ Spell.numTargets EQ1 | TargetValidTargeting$ Permanent.inRealZoneBattlefield,Player | Mode$ Gain | SubAbility$ DBCopy | StackDescription$ SpellDescription | SpellDescription$ Gain control of target spell that targets only a single permanent or player. Copy it, then reselect the targets at random for the spell and the copy. The new targets can't be you or a permanent you control.
-SVar:DBCopy:DB$ CopySpellAbility | Defined$ Targeted | RandomTarget$ True | RandomTargetRestriction$ Player.Other,Permanent.YouDontCtrl | SubAbility$ DBChangeTargets | StackDescription$ None
-SVar:DBChangeTargets:DB$ ChangeTargets | Defined$ Targeted | RandomTarget$ True | RandomTargetRestriction$ Player.Other,Permanent.YouDontCtrl
+A:SP$ ControlSpell | ValidTgts$ Card | TgtPrompt$ Select target spell that targets only a single permanent or player | TargetType$ Spell.numTargets EQ1 | TargetValidTargeting$ Permanent.inRealZoneBattlefield,Player | Mode$ Gain | RememberTargets$ True | SubAbility$ DBCopy | StackDescription$ SpellDescription | SpellDescription$ Gain control of target spell that targets only a single permanent or player. Copy it, then reselect the targets at random for the spell and the copy. The new targets can't be you or a permanent you control.
+SVar:DBCopy:DB$ CopySpellAbility | Defined$ Targeted | RememberCopies$ True | SubAbility$ DBChangeTargets | IgnoreFreeze$ True | StackDescription$ None
```

Comment:
Do you want RememberTargets _and_ RememberCopies?

Reply:
Need to change targets on both, but imo the Copy needs to be added to stack first (it matters for BecomesTarget triggers)